### ISSUE TITLE
RFC: 60 GHz for perleberger36<->strom-gw, perleberger36<->ilr

### ISSUE THREAD

**Issue by Noki** [freifunk-berlin/meta#134]
This Hardware requests might need some refinement based on input from location mainteners and should therefore be treated as a draft.

**Contact:**
@Noki 
@Akira25 

**Location:**
perleberger36
stom
ilr

**Use:**
We want to see more 60 GHz in the north of Berlin and think it is a good idea to start connecting gateways and locations that already have 60 GHz. To do so we want to place a Wave AP Micro at perleberger36 and connect strom-gw and ilr to it to bring 60 GHz to perleberger from 2 different directions, allowing us to extend the 60 GHz network from there in the future.

Akira25 and Noki will give 442 EUR to this cause.

**Devices List:**
perleberger36:
- Ubiquiti Wave Access Point Micro: ~520 EUR
ilr:
- Ubiquiti Wave Long-Range: ~300 EUR
strom:
- Ubiquiti UISP Wave Nano: ~302 EUR

**Total cost:**
1122 EUR - 442 EUR (Akira25 & Noki) = 680 EUR

**Checklist:**
_Internal for Stadtfunk, do not remove_

- [ ] Confirm decission
- [ ] Order devices and checkin (optional)
- [ ] Checkout devices
- [ ] Update budget

(See [instructions](https://github.com/freifunk-berlin/meta/blob/main/Hardware-requests.md))

**Comment by pmelange** [comment]
An important issue is to communicate our with to use 60ghz at strom-gw with inberlin.  They had objections to us using 60ghz in the past

**Comment by nicolasberens** [comment]
should i ask in-berlin about this?

**Comment by Noki** [comment]
> should i ask in-berlin about this?

Yes, please do so. If they have concerns about frequencies, we can simply set a different one from theirs.

**Comment by tscharly** [comment]
> I would suggest putting this building here into the planing: [www.openstreetmap.org/search?lat=52.538482&lon=13.341672&zoom=19#map=18/52.537469/13.342332](http://www.openstreetmap.org/search?lat=52.538482&lon=13.341672&zoom=19#map=18/52.537469/13.342332)

Why? Do you have access to it?
